src/docs/events.yaml

```diff
@@ -0,0 +1,204 @@
+paths:
+  /events/{id}:
+    get:
+      tags: 
+      - "Events"
+      summary: "Get event by id"
+      description: "Returns a single event"
+      operationId: "getEventById"
+      produces:
+      - "application/json"
+      parameters:
+        - name: "id"
+          in: "path"
+          description: "ID of the event to return"
+          required: true
+          type: "integer"
+          format: "int64"
+      responses:
+        "200":
+          description: "succesful operation"
+          schema:
+            $ref: "#/components/schemas/Event"
+        "404":
+          description: Invalid event ID supplied, Event not found
+        "500": 
+          description: Internal server error
+components:
+  schemas:
+    Event:
+     type: "object"
+     required:
+       - "title"
+       - "content"
+       - "coverImage"
+       - "date"
+       - "category"
+       - "address"
+       - "location"
+     properties:
+       publisherId:
+        type: "integer"
```

Comment:
Since this is populated, replace the type with: 
```
type: "object"
$ref: "#components/schemas/ShortUser"
```
What ShortUser is, is explained in the next comment